cmd/runtime/mkcert.go

```diff
@@ -19,6 +21,8 @@ var mkcertCmd = &cobra.Command{
 	Short: "Uses mkcert package to make locally-trusted development certificates.",
 	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		os.Setenv("TRUST_STORES", "nss")
```

Comment:
does this "add" to the defaults? or do we still have to add `system,java` somewhere else?

Reply:
this branch is just testing things on Fedora, but yeah. it's "setting" the stores. We can discuss how to choose which stores to use by the user or via config.